kak-lsp.toml

```diff
@@ -273,3 +273,26 @@ args = ["--stdio"]
 filetypes = ["zig"]
 roots = ["build.zig"]
 command = "zls"
+
+[language.julia]
```

Comment:
> Requires LanguageServer, StaticLint, SymbolServer installed globally in julia. 

What about adding something like `# Please install Julia packages "LanguageServer", "StaticLint" and "SymbolServer"` here. I haven't tried what happens if they aren't; maybe the error messages from kak-lsp are good enough, but I think they are only visible in the log.

Actually, would it make sense to install them automatically?
Since we are already running `julia`, we could add a snippet that runs the `Pkg.add` statements for missing packages.
Could that cause problems?

Reply:
(installing automatically is probably a bad idea)

I'm working on a patch that reports the stderr directly in Kakoune. Then we'll directly forward the error from julia which even tells the user how to install the package.